From f9c8bfbe6b104c774f2c9bcbb2026db5c24e70cc Mon Sep 17 00:00:00 2001
From: John Zhuge <jzhuge@cloudera.com>
Date: Wed, 3 May 2017 00:58:34 -0700
Subject: [PATCH 2398/2848] HDFS-11724. libhdfs compilation is broken on OS X.
 Contributed by John Zhuge.

(cherry picked from commit 343948ca795d20b0f77aa086f14e9a79d90a435b)

Conflicts:
	hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/libhdfs/os/posix/thread_local_storage.c

Change-Id: I054e80c0220d194aa077ce11566dabb07548a7ba
---
 .../native/libhdfs/os/posix/thread_local_storage.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/native/libhdfs/os/posix/thread_local_storage.c b/hadoop-hdfs-project/hadoop-hdfs/src/main/native/libhdfs/os/posix/thread_local_storage.c
index 9faa594..e6b59d6 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/native/libhdfs/os/posix/thread_local_storage.c
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/native/libhdfs/os/posix/thread_local_storage.c
@@ -19,7 +19,7 @@
 #include "os/thread_local_storage.h"
 
 #include <jni.h>
-#include <malloc.h>
+#include <stdlib.h>
 #include <pthread.h>
 #include <stdio.h>
 
-- 
1.7.9.5

